sprints/sprint_1.html

```diff
@@ -64,38 +64,6 @@ <h2>Results</h2>
       Listing of the rules is available in the description of the <a href="https://github.com/Neo23x0/sigma/pull/554">Pull Request</a> to Sigma repo.<br>
       Summary published on <a href="https://medium.com/oscd/oscd-threat-detection-sprint-1-c42317e06771">Medium</a> (EN) and <a href="https://habr.com/ru/post/482642/">Habr</a> (RU).
       <br><br>
```

Comment:
I believe that you can leave the list of participants for the sprint here as well, as it is only listing the participants of the particular sprint. please do the same for the sprint №2 page

Reply:
Done, for both sprints, thx